File: WorldFeed/Server/src/WorldFeed.Common/Public/Models/Interfaces/DTO/IMessageEventWithAppDTO.cs
namespace WorldFeed.Common.Public.Models.Interfaces.DTO
{
    using WorldFeed.Common.Public.Models.Interfaces.DTO.Events;

    /// <summary>
    /// DTO for encapsulating an MessageEvent and an App together for storage.
    /// Not used for transfer to or from Twitter.
    /// </summary>
    public interface IMessageEventWithAppDTO
    {
        IMessageEventDTO MessageEvent { get; set; }

        IApp App { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Public/Models/Interfaces/DTO/IMessageEventWithAppDTO.ts
import {IMessageEventDTO} from "./Events/IMessageEventDTO";
import {IApp} from "../IApp";

// DTO for encapsulating an MessageEvent and an App together for storage.
// Not used for transfer to or from Twitter.
export interface IMessageEventWithAppDTO {
  messageEvent: IMessageEventDTO;
  app: IApp;
}
